apache/trafficcontrol
# Logic to capture/display the last time a user successfully authenticated

## What does this PR (Pull Request) do?


- [x] This PR fixes #5412  


## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops
- Traffic Portal
- CI tests

## What is the best way to verify this PR?


1. Use GET curl calls for users/user api
```
https://<TO-API host:port>/api/4.0/user/current
OR 
https://<TO-API host:port>/api/4.0/users
OR 
https://<TO-API host:port>/api/4.0/users?id=<id>
```

One should see **last_authenticated** field in the response struct

2. Login to TP, go to Users section, click on select table column image, and you should see the `Last Authenticated` time if a user was successfully authenticated.
![image](https://user-images.githubusercontent.com/22248619/125345171-70ba2d80-e315-11eb-9050-f5a778b16e1e.png)

3. Run API/v4 tests for users

## If this is a bug fix, what versions of Traffic Control are affected?



## The following criteria are ALL met by this PR




## Additional Information
